### PR TITLE
Updated version and created new entry for service city zones API

### PR DESCRIPTION
**Pull request by ako977**

### CODE REVIEW

**Review by dschnare — COMMENTED** [review]
We don't ever expose our rate card correct?

File: specs/v1/paths/api/service-city-zones.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+get:
+  tags:
+    - Regions
+  summary: Retrieve service city zones
+  operationId: serviceCityGeographic areas
```
Comment (dschnare) [inline specs/v1/paths/api/service-city-zones.yaml:5]: Operation ID can't contain spaces can it?

**Review by dschnare — CHANGES_REQUESTED** [review]

File: specs/v1/paths/api/service-city-zones.yaml (hunk 1 above)
Comment (ako977) [reply specs/v1/paths/api/service-city-zones.yaml]: search replace error fixed

**Comment by ako977** [comment]
> We don't ever expose our rate card correct?

Correct, the props given are all described in the response structure

File: specs/v1/paths/api/service-city-zones.yaml (hunk 1 above)
Comment (ako977) [reply specs/v1/paths/api/service-city-zones.yaml]: > Operation ID can't contain spaces can it?

To answer the specific question though, it seems so, yes in the url it is changed to `%20`

**Pull request merged**